src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+function MoneyStringToNumber(string) {
+  return +string.slice(1).split(',').join('');
+}
+
+const peopleDataRows = document.querySelectorAll('tbody > tr');
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+const peopleSortedByName = [...peopleDataRows].sort(
+  (prevPerson, curPerson) => {
+    return prevPerson.children[0].textContent.localeCompare(
+      curPerson.children[0].textContent);
+  });
+const peopleSortedByPosition = [...peopleDataRows].sort(
+  (prevPerson, curPerson) => {
+    return prevPerson.children[1].textContent.localeCompare(
+      curPerson.children[1].textContent);
+  });
+const peopleSortedByAge = [...peopleDataRows].sort(
+  (prevPerson, curPerson) => {
+    return +prevPerson.children[2].textContent
+    - +curPerson.children[2].textContent;
+  });
+const peopleSortedBySalary = [...peopleDataRows].sort(
+  (prevPerson, curPerson) => {
+    return MoneyStringToNumber(prevPerson.children[3].textContent)
+    - MoneyStringToNumber(curPerson.children[3].textContent);
+  });
```

Comment:
no need to sort everything, just sort a particular column after a click, you can take by which column you should sort from 'event.target'.